tests/server.rs

```diff
@@ -132,6 +132,54 @@ async fn validate_wit_package(config: &Config, client: &FileSystemClient) -> Res
     Ok(())
 }
 
+async fn validate_content_policy(client: &FileSystemClient) -> Result<()> {
```

Comment:
This feels like it should just be part of the test since it performs a specific set of operations and validates them. Does it actually define any generic/shared logic?

Reply:
It's shared between the in-memory and postgres tests.

Reply:
From what I see, `it_rejects_non_wasm_content` is the only test that calls this function.

Reply:
Also called from function `test` in `tests/postgres/mod.rs` (one giant test case that tests against a single postgres server for reasons outlined in the comments).